testem.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  before_script: "npm run build",
```

Comment:
I vaguely think this will result in a double build (since the command to run is "npm run test" which does a build)